CMakeLists.txt

```diff
@@ -43,10 +43,11 @@ set(CYTHON_MIN_VERSION 0.25)
 
 # CMake policies
 foreach(pol
-        CMP0067  # honor language standard in try_compile()
-        CMP0071  # enable automoc for generated files
-        CMP0072  # prefers GLVND by default FindOpenGL
-       )
+		CMP0067  # honor language standard in try_compile()
```

Comment:
This were whitespaces before and now it's two tabs.

Reply:
i think the whitespaces are actually correct, as the entries should be aligned to `(pol`, and that alignment is relative to the start of `foreach`. that means the alignment is done with spaces just like the it was before :)

the alternative would be to place `pol` in the next line, where it would be `<tab>pol`, and all other entries would also be indented with one tab.